hack/bundle-automation/config.yaml

```diff
@@ -1,7 +1,7 @@
 
 - repo_name: multicloud-operators-subscription
   github_ref: "https://github.com/stolostron/multicloud-operators-subscription.git"
-  branch: "release-2.10"
+  branch: "main"
```

Comment:
Using `main` branch as reference until `release-2.11` branch is created for `multicloud-operators-subscription`.

Reply:
Verified that the `release-2.11` was created. Updated the config file to reflect those changes.